src/main/resources/hudson/tasks/Mailer/global_da.properties

```diff
@@ -25,9 +25,7 @@ E-mail\ Notification=E-mail p\u00e5mindelser
 Default\ user\ e-mail\ suffix=Standard e-mail endelse
 Use\ SSL=Brug SSL
 SMTP\ Port=SMTP Port
-Use\ SMTP\ Authentication=Benyt SMTP Authentificering
```

Comment:
@ndeloof `"${%Use SMTP Authentication}"` is still used in "Mailer/global.jelly".
Is it correct to move it from "Mailer/global*.properties" to "SMTPAuthentication/config*.properties"?